manifest.toml

```diff
@@ -1,84 +1,84 @@
 
-    [install.dns_over_https]
-    ask.en = "Should DNS-over-HTTPS be enabled? (If so, anyone who knows your adguard address can make a doh request to https://adguardomain.tld/dns-query)"
-    type = "boolean"
-    default = true
+[install.dns_over_https]
+ask.en = "Should DNS-over-HTTPS be enabled? (If so, anyone who knows your adguard address can make a doh request to https://adguardomain.tld/dns-query)"
+default = true
+type = "boolean"
 
 [resources]
-        [resources.sources.main]
-        in_subdir = "2"
-        arm64.url = "https://github.com/AdguardTeam/AdGuardHome/releases/download/v0.107.41/AdGuardHome_linux_arm64.tar.gz"
-        arm64.sha256 = "e23f5164a832c8647e0035b03189dc9c72821e9216ee5a5f8057739b8d072539"
-        amd64.url = "https://github.com/AdguardTeam/AdGuardHome/releases/download/v0.107.41/AdGuardHome_linux_amd64.tar.gz"
-        amd64.sha256 = "9e171202103ff57bd4ddebc280cbc71eb7a33f5b1ba23170f462b3dfcdca93f5"
-        armhf.url = "https://github.com/AdguardTeam/AdGuardHome/releases/download/v0.107.41/AdGuardHome_linux_armv7.tar.gz"
-        armhf.sha256 = "0ded3fcc63b009d7934730fd31e698b700153e6ac7dc205f5aa7352810a309d7"
-
-        autoupdate.strategy = "latest_github_release"
-        autoupdate.asset.amd64 = "^AdGuardHome_linux_amd64.tar.gz$"
-        autoupdate.asset.arm64 = "^AdGuardHome_linux_arm64.tar.gz$"
-        autoupdate.asset.armhf = "^AdGuardHome_linux_armv7.tar.gz$"
-
-    [resources.ports]
-    adguard.default = 53
-    adguard.exposed = "Both"
-
-    [resources.system_user]
-
-    [resources.install_dir]
-
-    [resources.permissions]
-    main.url = "/"
-    api.url = "/re:__DOMAIN__/dns-query"
-    api.allowed = "visitors"
-    api.auth_header = false
-    api.show_tile = false
-    api.protected = true
-
-    [resources.apt]
-    packages ="python3-bcrypt, python3-yaml"
+[resources.sources.main]
+amd64.sha256 = "204d0476f9dedb7bc7186c87fbf271c8cf952372372c0e5966e6ea932efe6d9d"
+amd64.url = "https://github.com/AdguardTeam/AdGuardHome/releases/download/v0.107.42/AdGuardHome_linux_amd64.tar.gz"
+arm64.sha256 = "48d1e518d1327e37bc542ca2b9242de91f0840bb4f4f37070f9d5990c9a90466"
+arm64.url = "https://github.com/AdguardTeam/AdGuardHome/releases/download/v0.107.42/AdGuardHome_linux_arm64.tar.gz"
+armhf.sha256 = "8f088cde491e5b154105002655d29bf4afd2467c4449f3cead10391f4399608c"
+armhf.url = "https://github.com/AdguardTeam/AdGuardHome/releases/download/v0.107.42/AdGuardHome_linux_armv7.tar.gz"
+in_subdir = 2
+
+autoupdate.asset.amd64 = "^AdGuardHome_linux_amd64.tar.gz$"
+autoupdate.asset.arm64 = "^AdGuardHome_linux_arm64.tar.gz$"
+autoupdate.asset.armhf = "^AdGuardHome_linux_armv7.tar.gz$"
+autoupdate.strategy = "latest_github_release"
+
+[resources.ports]
+adguard.default = 53
+adguard.exposed = "Both"
+
+[resources.system_user]
+
+[resources.install_dir]
+
+[resources.permissions]
+api.allowed = "visitors"
+api.auth_header = false
+api.protected = true
+api.show_tile = false
+api.url = "/dns-query"
```

Comment:
Randomly passing by : what this changes (compared to `"re:__DOMAIN__/dns-query"`) is that the new url is relative to `$domain/$path` of the app (so -> `$domain/$path/dns-query`). That doesnt change anything for full-domain install, but that's gonna break sub-path install if `/dns-query` is supposed to be an endpoint of the domain ...

However, the previous syntax sounds clearly broken : `api.url = "/re:__DOMAIN__/dns-query"` should have no `/` before the `re:` ...

The linter / schema may complain about this, but this is actually probably an error of the schema itself not covering those special kind of boring specific cases rather than something not supported by yunohost

Reply:
so the `re:` is here to make it non relative?

i trust you, make all the changes you want, I'm too tired to continue tonight sorry  :/